package.json

```diff
@@ -32,6 +32,7 @@
   },
   "devDependencies": {
     "@netlify/eslint-config-node": "^0.3.0",
+    "cpy": "^8.1.1",
```

Comment:
does cpx (aka what we already had installed) not work for this use case?

Reply:
Yes, it would. Fixed! :+1: